packages/react-fluentui-examples/List/list-active-element/src/List-Active-Element.tsx

```diff
@@ -0,0 +1,125 @@
+import {
+  Button,
+  makeStyles,
+  Persona,
+  mergeClasses,
+  Text,
+  tokens,
+} from '@fluentui/react-components';
+import { Mic16Regular } from '@fluentui/react-icons';
+import { List, ListItem } from '@fluentui/react-list-preview';
+
+import * as React from 'react';
+
+type Item = {
+  name: string;
+  id: string;
+  avatar: string;
+};
+
+const items: Item[] = [
+  'Melda Bevel',
+  'Demetra Manwaring',
+  'Eusebia Stufflebeam',
+  'Israel Rabin',
+  'Bart Merrill',
+  'Sonya Farner',
+  'Kristan Cable',
+].map((name) => ({
+  name,
+  id: name,
+  avatar:
+    'https://res-1.cdn.office.net/files/fabric-cdn-prod_20230815.002/office-ui-fabric-react-assets/persona-male.png',
+}));
+
+const useStyles = makeStyles({
+  selectedInfo: {
+    marginTop: '16px',
+  },
+  buttonWrapper: {
+    alignSelf: 'center',
+  },
+  item: {
+    cursor: 'pointer',
+    padding: '2px 6px',
+    justifyContent: 'space-between',
+    border: '1px solid transparent',
+  },
+  itemActive: {
+    backgroundColor: tokens.colorBrandBackground2,
+    border: `1px solid ${tokens.colorBrandStroke1}`,
+    borderRadius: '4px',
+  },
+});
+
+export const ListActiveElement = () => {
+  const classes = useStyles();
+
+  const [activeItem, setActiveItem] = React.useState<string | number | null>(
+    null
+  );
+
+  const onFocus = React.useCallback((event) => {
+    // Ignore bubbled up events from the children
+    if (event.target !== event.currentTarget) {
+      return;
+    }
+    setActiveItem(event.target.dataset.value);
+  }, []);
+
+  const onAction = React.useCallback((_, { value }) => {
+    setActiveItem(value);
+  }, []);
+
+  return (
+    <div>
+      <List navigationMode="composite">
+        {items.map(({ name, avatar }) => (
+          <ListItem
+            key={name}
+            value={name}
+            className={mergeClasses(
+              classes.item,
+              activeItem === name && classes.itemActive
```

Comment:
should there also be aria-selected or some other aria prop indicating the selection state?